azurerm/config.go

```diff
@@ -534,6 +539,16 @@ func (c *ArmClient) registerContainerServicesClients(endpoint, subscriptionId st
 	c.kubernetesClustersClient = kubernetesClustersClient
 }
 
+func (c *ArmClient) registerDatabricksClients(endpoint, subscriptionId string, auth autorest.Authorizer, sender autorest.Sender) {
+	databricksWorkspacesClient := databricks.NewWorkspacesClientWithBaseURI(endpoint, subscriptionId)
+	setUserAgent(&databricksWorkspacesClient.Client)
```

Comment:
Is it possible to replace these lines of code with [`configureClient()`](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/config.go#L212) like [the example here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/config.go#L521).